src/geometries/ExtrudeGeometry.js

```diff
@@ -439,6 +439,24 @@ class ExtrudeGeometry extends BufferGeometry {
 
 			}
 
+			//When bevelSegments == 0, the modified vert arrays will not have been populated. We do this here.
+
+			if ( contractedContourVertices.length == 0 ) {
+
+				contractedContourVertices.push( ...contour );
```

Comment:
I've seen the spread operator cause stackoverflow errors when used with "push" - lets avoid using it here. Is there a reason to push the vertices onto the array rather than replacing the variable?